src/time/time_format/formatter.cr

```diff
@@ -176,6 +176,101 @@ struct TimeFormat
       {negative, hours, minutes}
     end
 
+    # Compute week of the year, monday is the first day of the week
+    def week_of_year_monday_1_7
+      io << Formatter.iso8601_week_of_year(time, 1)
+    end
+
+    # Compute week of the year, sunday is the first day of the week
+    def week_of_year_sunday_0_6
+      io << Formatter.iso8601_week_of_year(time, 0)
+    end
+
+    # Internal helper wethod: Figure how many weeks into the year
+    def self.week_of_year(time, firstweekday)
```

Comment:
> Personally I also prefer code being called to reside below code calling it.

You're right, I'll swap it.

> Maybe make it private?

I'll doo. Just a question regarding that: Why is `private self.method_name` not possible? And whats the correct way to define a static private method in crystal?